pom.xml

```diff
@@ -28,8 +28,8 @@ THE SOFTWARE.
    <parent>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>plugin</artifactId>
-      <version>1.424.6</version>
-      <relativePath />
+      <version>1.609.3</version>
```

Comment:
BTW consider using the 2.x parent POM at some point. A lot more pleasant to work with. (The baseline would in that case be defined using the `jenkins.version` property.)